shiny-app/config.yml

```diff
@@ -1,8 +1,11 @@
 default:
   db:
     drv: !expr RSQLite::SQLite()
-    dbname: app.db
-
+    dbname: "../portfolio.db"
+quickstart:
+  db:
+    drv: !expr RSQLite::SQLite()
+    dbname: "/data/portfolio.db"
 colorado:
```

Comment:
We should remove these! My bad for having it included originally :/

Reply:
No worries! I was thinking it would be good to have the template in case somebody wanted to deploy to Colorado and change the parameters to point at the colorado database. No problem to remove either, though!